src/model/ButtonVersion.ts

```diff
@@ -1,6 +1,13 @@
 /**
  * @title Тема кнопки
- * @enumNames ["Основная", "Второстепенная", "Белая", "Ссылка", "Прозрачная", "Наследуемая"]
+ * @enumNames ["Основная", "Второстепенная", "Белая", "Ссылка", "Прозрачная", "Серая", Наследуемая"]
```

Comment:
```suggestion
 * @enumNames ["Основная", "Второстепенная", "Белая", "Ссылка", "Прозрачная", "Серая", "Наследуемая"]
```

уже писали про недостающую кавычку и поломанную генерацию схемы.